packages/ui/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@webpack-cli/ui",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
```suggestion
```

Reply:
Will do separately as `ui` is moved to `ui-gui` 😅